docs-src/concepts/what-is-a-worker-process.md

```diff
@@ -21,7 +21,7 @@ More formally, a Worker Process is any process that implements the Task Queue Pr
 
 **Worker Processes are external to a Temporal Cluster.**
 Temporal Application developers are responsible for developing [Worker Programs](/concepts/what-is-a-worker-program) and operating Worker Processes.
-Said another way, the [Temporal Cluster](/concepts/what-is-a-temporal-cluster) (including the Temporal Cloud) doesn't execute any of your code (Workflow and Activity Definitions) on Temporal Cluster machines. The Cluster is solely responsible for orchestrating state transitions and providing Tasks to the next available Worker Entity.
+Said another way, the [Temporal Cluster](/concepts/what-is-a-temporal-cluster) (including the Temporal Cloud) doesn't execute any of your code (Workflow and Activity Definitions) on Temporal Cluster machines. The Cluster is solely responsible for orchestrating [State Transitions](/concepts/what-is-a-state-transition) and providing Tasks to the next available Worker Entity.
```

Comment:
```suggestion
Said another way, the [Temporal Cluster](/concepts/what-is-a-temporal-cluster) (including the Temporal Cloud) doesn't execute any of your code (Workflow and Activity Definitions) on Temporal Cluster machines. The Cluster is solely responsible for orchestrating [State Transitions](/concepts/what-is-a-state-transition) and providing Tasks to the next available Worker.
```
Can maybe drop "Entity"?

Reply:
@flossypurse: Given the number of instances of "Worker Entit[y|ies]" in this topic, I'm inclined to link this first instance to `/concepts/what-is-a-worker-entity`.